app/src/ereferrals/java/org/eyeseetea/malariacare/domain/usecase/DownloadLanguageTranslationUseCase.java

```diff
@@ -3,20 +3,23 @@
 
 import org.eyeseetea.malariacare.data.authentication.CredentialsReader;
 import org.eyeseetea.malariacare.data.sync.importer.strategies.ILanguagesClient;
-import org.eyeseetea.malariacare.data.sync.importer.strategies.LanguageDownloader;
+import org.eyeseetea.malariacare.data.sync.importer.LanguageDownloader;
 import org.eyeseetea.malariacare.domain.boundary.IConnectivityManager;
 import org.eyeseetea.malariacare.domain.boundary.executors.IAsyncExecutor;
+import org.eyeseetea.malariacare.domain.boundary.repositories.ISettingsRepository;
 import org.eyeseetea.malariacare.locale.factory.LanguageFactory;
 
 public class DownloadLanguageTranslationUseCase implements UseCase {
 
     private IConnectivityManager connectivity;
     private CredentialsReader credentialsReader;
+    private ISettingsRepository mSettingsRepository;
 
     public DownloadLanguageTranslationUseCase(CredentialsReader credentialsReader,
-            IConnectivityManager connectivity) {
+            IConnectivityManager connectivity,ISettingsRepository settingsRepository) {
```

Comment:
Pass settingsRepository as argument to DownloadLanguageTranslationUseCase is not necessary